CONTRIBUTING.md

```diff
@@ -92,3 +92,34 @@ For readmes and other documentation written in Markdown, [these rules](https://g
 Similar to C++ these rules can be adapted to the needs of the individual repository with a markdownlint.json file.
 You can find an example [here](https://github.com/openMSL/sl-1-0-sensor-model-repository-template/blob/main/.github/workflows/markdownlint.json).
 The style of all .md files will be checked against these config file in the CI pipeline of the repository.
+
+## Implement Changes
+
+In order to implement changes in an OpenMSL repository, please follow the process: issue -> branch -> draft pull request -> pull request.
```

Comment:
branch is only possible with write access. Therefore it would technically be a fork. We can encourage people who intent to make changes more often to request write access as regular contributor and make a group  for this. I think Issues are only necessary if there is somethin unclear or just a report of an issue or dropping an idea. In general people should contribute a PR right away if they think they know the solution already

Reply:
I will add, that people can fork and create a branch or alternatively request access to the contributor group.

To my knowledge it is the standard git workflow to create an issue first and every PR closes an issue. I usually create the branch directly from the issue in the right menu.

Reply:
I am fine with this as well. We can add a note that a solution is welcome. Otherwise the issue section is always like a wish list with the expectation that others will solve their problems :D